packages/sso-menu/src/SSOMenu.tsx

```diff
@@ -0,0 +1,248 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Button from '@leafygreen-ui/button';
+import Icon from '@leafygreen-ui/icon';
+import { Menu, MenuItem, MenuGroup } from '@leafygreen-ui/menu';
+import { uiColors } from '@leafygreen-ui/palette';
+import { css, cx } from '@leafygreen-ui/emotion';
+
+const menuButtonStyle = css`
+  height: 29px;
+  padding: 2px 14px;
+  border: 1px solid ${uiColors.gray.base};
+  border-radius: 14.5px;
+  cursor: pointer;
+  transition: background 200ms ease-in-out;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: ${uiColors.gray.dark1};
+  font-size: 12px;
+  line-height: 15px;
+  position: relative;
+  &:hover {
+    background-color: ${uiColors.gray.light1};
+    color: ${uiColors.gray.dark2};
+  }
+`;
+
+const menuNameStyle = css`
+  margin-right: 2px;
+  margin-left: 2px;
+`;
+
+const activeMenuButtonStyle = css`
+  background-color: ${uiColors.gray.base};
+  color: ${uiColors.white};
+  &:hover {
+    background-color: ${uiColors.gray.base};
+    color: ${uiColors.white};
+  }
+`;
+
+const nameStyle = css`
+  font-size: 16px;
+  color: ${uiColors.gray.dark1};
+  font-weight: bolder;
+  margin-top: 4px;
+  margin-bottom: 2px;
+  margin-right: 0px;
+  margin-left: 0px;
+`;
+
+const truncate = css`
+  width: 162px;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+`;
+
+const iconStyle = css`
+  position: relative;
+  fill: ${uiColors.gray.dark1} !important;
+  transition: color 200ms ease-in-out;
+`;
+
+const accountMenuGroupStyle = css`
+  padding: 20px 20px 14px;
+`;
+
+const accountButtonStyle = css`
+  margin-top: 12px;
+  width: 100%;
+  display: inline-flex;
+  justify-content: center;
+`;
+
+const menuItemTextStyle = css`
+  font-size: 14px;
+  line-height: 16px;
+  cursor: pointer;
+  margin: 0px;
+`;
+
+const descriptionStyle = css`
+  margin: 0px;
+  display: block;
+  font-weight: normal;
+  font-size: 12px;
+  color: ${uiColors.gray.dark2};
+  text-decoration: none;
+`;
+
+const menuItemPadding = css`
+  padding-left: 20px;
+`;
+
+const logoutContainerHeight = css`
+  height: 46px;
+`;
+
+const menuItems = [
+  {
+    displayName: 'Atlas',
+    description: 'cloud.mongodb.com',
+    href: 'https://cloud.mongodb.com',
+    slug: 'atlas',
+  },
+  {
+    displayName: 'University',
+    description: 'university.mongodb.com',
+    href: 'https://university.mongodb.com',
+    slug: 'university',
+  },
+  {
+    displayName: 'Cloud Support',
+    description: 'support.mongodb.com',
+    href: 'https://support.mongodb.com',
+    slug: 'support',
+  },
+] as const;
+
+const slugs = menuItems.map(mi => mi.slug);
+
+export const ActiveProduct = {
```

Comment:
If we're exporting this, the name doesn't seem appropriate as it doesn't tell you which one is active.

Reply:
ActiveProductOptions?

Reply:
Why not "Products" or "ProductOptions"? I don't think I understand why it says "Active".

Reply:
Done!